android-metadata-generator/src/src/com/telerik/metadata/Builder.java

```diff
@@ -60,7 +60,11 @@ public static TreeNode build(String[] paths) throws Exception {
                 // our class path API 17
                 // Class<?> clazz = Class.forName(className, false, loader);
                 ClassDescriptor clazz = ClassRepo.findClass(className);
-                generate(clazz, root);
+                if (clazz == null) {
+                    throw new ClassNotFoundException("Class " + className + " not found in the input jars.");
```

Comment:
Change jars to libraries, since from the user perspective `.aar` files are not jars.